web/src/compose_recipient.ts

```diff
@@ -356,14 +369,15 @@ export function update_topic_displayed_text(
     // reset
     $input.attr("placeholder", "");
     $input.removeClass("empty-topic-display");
-    $topic_not_mandatory_placeholder.css({visibility: "hidden"});
+    $topic_not_mandatory_placeholder.removeClass("visible");
     $topic_not_mandatory_placeholder.hide();
 
     function update_placeholder_visibility(): void {
-        $topic_not_mandatory_placeholder.css(
-            "visibility",
-            $input.val() === "" ? "visible" : "hidden",
-        );
+        if ($input.val() === "") {
+            $topic_not_mandatory_placeholder.addClass("visible");
+        } else {
+            $topic_not_mandatory_placeholder.removeClass("visible");
+        }
```

Comment:
We have this meme all over the place, but it’s [unnecessary](https://api.jquery.com/toggleClass/):

```js
$topic_not_mandatory_placeholder.toggleClass("visible", $input.val() === "");
```

Reply:
PR #34202